{
    "track_id": "lmaeKuvR",
    "prompt": "Unknown ERROR->NameError",
    "time": "2022-03-18 12:45:57",
    "content": "Traceback (most recent call last):\n  File \"/root/Projects/Kori-Bot/Kori-bot/./ATRI/exceptions.py\", line 102, in _track_error\n    raise exception\n  File \"/usr/local/lib/python3.8/dist-packages/nonebot/message.py\", line 202, in _run_matcher\n    await matcher.run(bot, event, state, stack, dependency_cache)\n  File \"/usr/local/lib/python3.8/dist-packages/nonebot/internal/matcher.py\", line 668, in run\n    await self.simple_run(bot, event, state, stack, dependency_cache)\n  File \"/usr/local/lib/python3.8/dist-packages/nonebot/internal/matcher.py\", line 635, in simple_run\n    await handler(\n  File \"/usr/local/lib/python3.8/dist-packages/nonebot/dependencies/__init__.py\", line 92, in __call__\n    return await self.call(**values)\n  File \"/root/Projects/Kori-Bot/Kori-bot/./src/plugins/songpicker2/__init__.py\", line 39, in handle_songName\n    songName = state[\"songName\"]\nNameError: name 'state' is not defined\n"
}